miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -172,14 +180,14 @@ int main(int argc, char *argv[])
    args.AddOption(&normalization, "-nor", "--normalization", "-no-nor",
                   "--no-normalization",
                   "Make all terms in the optimization functional unitless.");
-   args.AddOption(&fdscheme, "-fd", "--fd_approximation",
+   args.AddOption(&fdscheme, "-fd", "--fd_approximation", "no-fd", "no-fd-app",
```

Comment:
`"no-fd"` --> `"-no-fd"`
`"no-fd-app"` --> `"--no-fd_approximation"` (or use `-` instead of `_` in both the enable and disable versions)